v01/http.go

```diff
@@ -97,8 +97,8 @@ func (e *Event) UnmarshalBinary(req *http.Request) error {
 			continue
 		}
 
-		k = strings.TrimLeft(k, "Ce-")
-		k = strings.TrimLeft(k, "X-")
+		k = strings.TrimPrefix(k, "Ce-")
+		k = strings.TrimPrefix(k, "X-")
 		k = strings.Replace(k, "-", "", -1)
```

Comment:
why do this? isn't this buggy since it would make "foo-bar" overwrite "foobar" ?

Reply:
I think we can safely remove this because we decided that property names are just lowercase ASCII strings.